config/manager/kustomization.yaml

```diff
@@ -12,4 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/bpradipt/cc-operator
+  newName: quay.io/jensfr/confidential-containers-operator
```

Comment:
How about changing it to generic values ?
```
  newName: controller
  newTag: latest
```

Reply:
yes, of course, good catch, that shouldn't be in there. thanks!

Reply:
I changed this also